website/views/layouts/layout.ejs

```diff
@@ -297,10 +297,14 @@
         function windowScrolled() {
             var scrollTop = window.pageYOffset || document.documentElement.scrollTop
             var fleetLogo = header.querySelector('img')
+            var hamburgerMenuIcon = header.querySelector('.btn.btn-link > img')
+            console.log(hamburgerMenuIcon)
```

Comment:
was this `console.log` commited on purpose?

Reply:
Thanks for that catch @roperzh. It wasn't!